packages/api-client-core/src/GadgetFunctions.ts

```diff
@@ -108,11 +108,12 @@ export interface ActionFunctionMetadata<OptionsT, VariablesT, SelectionT, Schema
   variablesType: VariablesT;
   isBulk: IsBulk;
   hasAmbiguousIdentifier?: boolean;
-  /** @deprecated */
-  hasCreateOrUpdateEffect?: boolean;
   acceptsModelInput?: boolean;
   paramOnlyVariables?: readonly string[];
   hasReturnType?: boolean;
+  singleActionFunctionName?: string;
```

Comment:
I don't see this one used anywhere in this PR, but it's indeed in there, so game for fully specifying the shape of this.